### ISSUE TITLE
Design | Migrate to Figma | Analyze Sketch Files for Exporting and Archiving

### ISSUE THREAD

**Issue by mtcA6** [department-of-veterans-affairs/va.gov-team#71216]
## Background 
We need to migrate the sketch files to Figma and in order to do that, we need to have our sketch files updated to the latest USWDS design components. We are not going to migrate ALL of the sketch files that we have. 

The goal of this ticket is to work through all of the profile team's sketch files to see what items need to be exported and archived and not converted to sigma. Once sketch licensing ends we'll lose access to the sketch files and we should preserve those in some way even if we aren't migrating them. 

## Tasks 
***check to make sure that I've not missed a section or sketch file that should be referenced 
- [ ] check the following sketch files 
     - [ ] nametag 
     - [ ] hub 
     - [ ] personal information 
     - [ ] contact information 
     - [ ] military information 
     - [ ] direct deposit 
     - [ ] notification preferences 
     - [ ] account security 
     - [ ] connected apps  
     - [ ] editing as a subtask 
     - [ ] personal health contacts 

## Acceptance Criteria 
- [ ] we have a list of the files/sketch urls that we are not migrating but need to export and save/archive in our github directory

**Comment by pamela-drouin** [comment]
@andaleliz I have been poking at this ticket, and I have a pretty solid overview of the composition of each Sketch file. You're probably already very familiar with this!

- Nearly all Profile Sketch files have `Mobile` and `Desktop` pages.  
- Many have `User flow`. 
- Many of these files have some variation of `Exploration`, `Future state`, `Iterations`, and `Discovery`. 
- A few of them have `Specs` or `Style guide`. 
- There are also assorted one-off pages.

It's a given to me that `Mobile`, `Desktop`, and `User flow` pages are kept/migrated. I'm less sure about the other common pages or the one-off pages. I would love to know your thoughts!

**Comment by andaleliz** [comment]
@pamela-drouin user flow pages can actually be deleted; those were what we used before we went to the annotated versions of user flows we worked on earlier this year.

- I think personal info is the only file with `Future State`, and we should keep that because it has all the design work for birth sex and sexual orientation, when we get to add those to the page.
- A previous designer and I created the iterations page to document past versions of the page, especially if there were significant design changes. I'm kind of on the fence about whether or not those are useful to keep around. What do you think?
- I think `discovery` and `explorations` can probably be archived. Maybe deleted or if for some reason you see anything worthwhile, we could export and then save it in GitHub or something.
- `Specs` and `Style guide` are probably only in notification settings, I think we should put those in Figma for now. I know they were helpful to Adam and if we'd like to follow that pattern for future things, it'd be good to be able to reference them.
- I think assorted one off pages can be deleted, except the military info things since we're actively working on that.

If there's anything you feel strongly about deleting or keeping that's different from what I suggested, that's ok too!

**Comment by pamela-drouin** [comment]
Thanks, @andaleliz! Re: iteration pages -- since we're already archiving some pages, I lean towards archiving these too, just in case 🤓 

Here's what I have so far -- I'm not sure how Figma handles the Symbols pages -- do we migrate those? Or should we archive/delete? 

- `Keep: ✅`;  `Archive: 📦`; `Trash: 🗑`; `Unsure:❓`
	- [ ] My VA and Profile - Nametag
		- [ ] ✅ Known use cases 
	- [ ] Profile - Hub landing page
		- [ ] ✅ Mobile 
		- [ ] ✅ Desktop 
		- [ ] 📦 Exploration: Long term plan for alerts 
		- [ ] ❓Symbols 
	- [ ] Profile - Personal information
		- [ ] ✅ Mobile 
		- [ ] ✅ Desktop 
		- [ ] COLA Maintenance Window - `#`69385
		- [ ] 📦 Exploration - add CST link
		- [ ] ✅ Future state - Mobile
		- [ ]  ✅ Future state - Desktop
		- [ ] Symbols
		- [ ] 📦 Discovery
		- [ ] 📦 Iterations
	- [ ] Profile - Contact Information
		- [ ] ✅ Mobile 
		- [ ] ✅ Desktop 
		- [ ] 📦 Iterations/discovery - modal and BAI
		- [ ] 📦 Exploration- address validation accessibility improvement
		- [ ] Symbols
		- [ ] 📦 Iterations - Launched versions
	- [ ] Profile - Military Information
		- [ ] ✅ Mobile 
		- [ ] ✅ Desktop 
		- [ ] `#`68362 - Enhanced military info
		- [ ] 📦 Iterations
		- [ ] 🗑 Page 1 
		- [ ] 📦 Discovery
		- [ ] Symbols
	- [ ] Profile - Direct Deposit
		- [ ] ✅ Mobile
		- [ ] `#`69385 - COLA Maintenance Window
		- [ ] ✅ Desktop
		- [ ] `#`63493 - a11y fraud note improvement
		- [ ] Supporting content pages
		- [ ] Incident Alerts
		- [ ] 📦 Explorations
		- [ ] Symbols
		- [ ] 📦 Discovery
		- [ ] 📦 Iterations
	- [ ] Profile - Notification Settings
		- [ ] ✅ Mobile 
		- [ ] ✅ Desktop
		- [ ] 📦 Exploration - update help text in select screens
		- [ ] 📦 Exploration - update missing contact info alert text
		- [ ] 📦 Exploration - Add 1095-B form
		- [ ] ✅ Specs
		- [ ] Symbols
		- [ ] 📦 Iterations
	- [ ] Profile - Account Security
		- [ ] ✅ Mobile 
		- [ ] ✅ Desktop
		- [ ] Symbols
		- [ ] 📦 Iterations
		- [ ] 📦 Discovery
		- [ ] 📦 Archive
	- [ ] Profile - Connected Apps
		- [ ] ✅ Mobile 
		- [ ] ✅ Desktop
		- [ ] Symbols
	- [ ] Profile - Editing sub-task
		- [ ] ✅ Mobile 
		- [ ] ✅ Desktop
		- [ ] 📦 Iterations 
		- [ ] 🗑 User flow
		- [ ] Symbols
	- [ ] Profile - Personal health care contact
		- [ ] 🗑 User flow (should be in mobile)
		- [ ] Hub landing page
		- [ ] ✅ Mobile - Emergency contact and Next of kin
		- [ ] ✅ Desktop - Emergency contact and Next of kin
		- [ ] For review
		- [ ] `#`64873 - Exploration -  Next of kin and Emergency contact  
		- [ ] Symbols
		- [ ] 📦 Iterations
	- [ ] Profile - Audit of alert messages
		- [ ] ✅ Inventory_Audit 
		- [ ] Symbols
		- [ ] 🗑 graveyard
	- [ ] Deprecated -Profile-2.0
		- [ ] Screens 1.0
		- [ ] Screens 2.0  pre-launch Sept 2020
		- [ ] Screens 2.0 Mobile exploration
		- [ ] Style Guide
		- [ ] Symbols

**Comment by andaleliz** [comment]
Wow that's a lot of pages! The symbols pages do migrate over, so I think those are good to keep! I agree with all the icon-ed items in your list (also that is a great way to track them!). 
- We might need to incorporate the ticket number direct deposit pages into the mobile/desktop pages, if they're on production now. Maybe we need a separate ticket to look into that if its something that'll take more than a few minutes?
- I think we should archive the 2.0 file, just for historical purposes.
- I'm not sure about anything in the personal health care contacts page, since Jonathan is the owner of that

**Comment by pamela-drouin** [comment]
@jonathanjnelson Hi, Jonathan! What are your plans for the Profile - Personal health care contact sketch file? I'm working on this ticket, but I won't be touching your file till I hear your plans 😄

**Comment by pamela-drouin** [comment]
Latest update @mtcA6 @andaleliz! This is nearly done, with the exception of Military Information, which is currently being worked on. 

I have consolidated ad hoc page content into Exploration pages, and deleted any pages that won't be migrated or archived.

Maybe we can write a separate ticket for exporting/archiving?

Without further ado, here's the list of pages to migrate/archive.

KEY: `Keep: ✅`;  `Archive: 📦`; `Trash: 🗑`; `Relocate: ➡️`

- List of Sketch files and their pages
	- [ ] Profile - Personal information
		- [ ] ✅ Mobile 
		- [ ] ✅ Desktop 
		- [ ] ✅ Future state - Mobile
		- [ ] ✅ Future state - Desktop
		- [ ] ✅ Symbols
		- [ ] 📦 Discovery
		- [ ] 📦 Iterations
	- [ ] Profile - Military Information `On hold: currently being used`
		- [ ] ✅ Mobile 
		- [ ] ✅ Desktop 
		- [ ] ✅ Symbols
		- [ ] 📦 Discovery
		- [ ] 📦 Iterations
		- [ ] `#`68362 - Enhanced military info - Lo Fi
		- [ ] `#`71889 - Enhanced military info - Hi fi
		- [ ] `#`71889 - Explorations Enhanced military info - Hi fi
	- [ ] Profile - Direct Deposit
		- [ ] ✅ Mobile
		- [ ] ✅ Desktop
		- [ ] ✅ Symbols
		- [ ] 📦 Discovery
		- [ ] 📦 Explorations
		- [ ] 📦 Iterations
	- [ ] Profile - Editing sub-task
		- [ ] ✅ Mobile 
		- [ ] ✅ Desktop
		- [ ] ✅ Symbols
		- [ ] 📦 Iterations 
	- [ ] My VA and Profile - Nametag
		- [ ] ✅ Known use cases 
	- [ ] Profile - Hub landing page
		- [ ] ✅ Mobile 
		- [ ] ✅ Desktop 
		- [ ] ✅ Symbols 
		- [ ] 📦 Exploration: Long term plan for alerts 
	- [ ] Profile - Contact Information
		- [ ] ✅ Mobile 
		- [ ] ✅ Desktop 
		- [ ] ✅ Symbols
		- [ ] 📦 Iterations - Launched versions
		- [ ] 📦 Iterations/discovery - modal and BAI
		- [ ] 📦 Exploration- address validation accessibility improvement
	- [ ] Profile - Notification Settings
		- [ ] ✅ Mobile 
		- [ ] ✅ Desktop
		- [ ] ✅ Symbols
		- [ ] ✅ Specs
		- [ ] 📦 Iterations
		- [ ] 📦 Exploration - update help text in select screens
		- [ ] 📦 Exploration - update missing contact info alert text
		- [ ] 📦 Exploration - Add 1095-B form
	- [ ] Profile - Account Security
		- [ ] ✅ Mobile 
		- [ ] ✅ Desktop
		- [ ] ✅ Symbols
		- [ ] 📦 Iterations
		- [ ] 📦 Discovery
		- [ ] 📦 Archive
	- [ ] Profile - Connected Apps
		- [ ] ✅ Mobile 
		- [ ] ✅ Desktop
		- [ ] ✅ Symbols
	- [ ] Profile - Personal health care contact
		- [ ] ✅ Mobile - Emergency contact and Next of kin
		- [ ] ✅ Desktop - Emergency contact and Next of kin
		- [ ] ✅ Symbols
		- [ ] 📦 Iterations
		- [ ] 📦  Explorations
	- [ ] Profile - Audit of alert messages 📦
		- [ ] 📦 Inventory_Audit 
		- [ ] 📦 Symbols
	- [ ] Deprecated -Profile-2.0 📦 
		- [ ] 📦 Screens 1.0
		- [ ] 📦 Screens 2.0  pre-launch Sept 2020
		- [ ] 📦 Screens 2.0 Mobile exploration
		- [ ] 📦 Style Guide
		- [ ] 📦 Symbols

**Comment by andaleliz** [comment]
@pamela-drouin I'm working on the account security files now, and after looking at the discovery and archive pages, I think we can trash them.  The previous versions of this page are never coming back :)

**Comment by mtcA6** [comment]
@pamela-drouin updated all of the corresponding tickets. Are we okay to close this story? 

Based on Liz's comment from last week, I think the next to focus on would be:
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/72570

What I can't tell, since I don't have access to figma, is if Liz was migrating based on that list you layed out. I'm not sure what your plan was for #72331 but I'm thinking we should assign and work on that one last as a way to review and ensure we've archived all of the pages you originally listed.  

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/72331

**Comment by andaleliz** [comment]
@mtcA6 the ticket you highlighted as the next one to focus on is blocked by #72127, so we can't move forward with that until the library is moved over.

**Comment by pamela-drouin** [comment]
@mtcA6 Yeah, this ticket is good to close out!
